templates/macros/actions.html.twig

```diff
@@ -23,6 +23,12 @@
             {% endif %}
             {% set actions = actions|merge({'edit': {'url': path('admin_activity_edit', {'id': activity.id}), 'class': class}}) %}
         {% endif %}
+        {% if is_granted('view_other_timesheet') %}
+            {% set actions = actions|merge({'timesheet': path('admin_timesheet', {'customer': activity.project ? activity.project.customer.id : null, 'project': activity.project ? activity.project.id : null, 'activity': activity.id})}) %}
```

Comment:
what happens to the URL if you pass null for a parameter?

Reply:
@kevinpapst As far as I tested it the parameter is ignored and the URL doesn't contain it.